WARNING: Ignoring invalid distribution -erminado (/mnt/home/n9455647/.conda/envs/conda_test/lib/python3.8/site-packages)
WARNING: Ignoring invalid distribution -cipy (/mnt/home/n9455647/.conda/envs/conda_test/lib/python3.8/site-packages)
/home/n9455647/.conda/envs/conda_test/lib/python3.8/site-packages/pyAgrum/lib/notebook.py:1312: UserWarning: 
  ** pyAgrum.lib.notebook has to be imported from an IPython's instance (mainly notebook).
  
  warnings.warn("""
RuntimeError: module compiled against API version 0xf but this version of numpy is 0xe
  0%|          | 0/14 [00:00<?, ?it/s]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [02:27<17:12, 147.45s/it][A
 25%|██▌       | 2/8 [04:50<14:30, 145.13s/it][A
 38%|███▊      | 3/8 [07:06<11:44, 140.93s/it][A
 50%|█████     | 4/8 [09:24<09:17, 139.50s/it][A
 62%|██████▎   | 5/8 [11:46<07:02, 140.68s/it][A
 75%|███████▌  | 6/8 [13:53<04:31, 135.87s/it][A
 88%|████████▊ | 7/8 [15:55<02:11, 131.48s/it][A
100%|██████████| 8/8 [18:11<00:00, 132.64s/it][A100%|██████████| 8/8 [18:11<00:00, 136.38s/it]
  7%|▋         | 1/14 [18:13<3:56:50, 1093.08s/it]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [04:08<28:58, 248.38s/it][A
 25%|██▌       | 2/8 [08:16<24:50, 248.38s/it][A
 38%|███▊      | 3/8 [12:24<20:41, 248.26s/it][A
 50%|█████     | 4/8 [16:33<16:34, 248.52s/it][A
 62%|██████▎   | 5/8 [20:36<12:19, 246.36s/it][A
 75%|███████▌  | 6/8 [24:45<08:14, 247.34s/it][A
 88%|████████▊ | 7/8 [28:39<04:02, 242.93s/it][A
100%|██████████| 8/8 [32:28<00:00, 238.49s/it][A100%|██████████| 8/8 [32:28<00:00, 243.55s/it]
 14%|█▍        | 2/14 [50:47<5:19:57, 1599.77s/it]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [05:02<35:17, 302.45s/it][A
 25%|██▌       | 2/8 [10:22<31:17, 312.89s/it][A
 38%|███▊      | 3/8 [19:04<34:00, 408.10s/it][A
 50%|█████     | 4/8 [34:27<40:46, 611.74s/it][A
 62%|██████▎   | 5/8 [43:28<29:17, 585.95s/it][A
 75%|███████▌  | 6/8 [51:53<18:36, 558.47s/it][A
 88%|████████▊ | 7/8 [1:03:25<10:02, 602.20s/it][A
100%|██████████| 8/8 [1:12:16<00:00, 579.61s/it][A100%|██████████| 8/8 [1:12:16<00:00, 542.10s/it]
 21%|██▏       | 3/14 [2:03:10<8:42:53, 2852.15s/it]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [12:31<1:27:39, 751.32s/it][A
 25%|██▌       | 2/8 [25:27<1:16:34, 765.72s/it][A
 38%|███▊      | 3/8 [32:58<51:51, 622.25s/it]  [A
 50%|█████     | 4/8 [40:39<37:13, 558.35s/it][A
 62%|██████▎   | 5/8 [8:28:26<8:43:54, 10478.27s/it][A
 75%|███████▌  | 6/8 [8:41:12<3:59:12, 7176.10s/it] [A
 88%|████████▊ | 7/8 [8:53:41<1:24:35, 5075.08s/it][A
100%|██████████| 8/8 [9:09:48<00:00, 3767.18s/it]  [A100%|██████████| 8/8 [9:09:48<00:00, 4123.57s/it]
 29%|██▊       | 4/14 [11:13:06<40:58:49, 14752.92s/it]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [27:41<3:13:52, 1661.79s/it][A
 25%|██▌       | 2/8 [37:19<1:42:25, 1024.21s/it][A
 38%|███▊      | 3/8 [46:58<1:08:24, 820.80s/it] [A
 50%|█████     | 4/8 [1:05:55<1:03:03, 945.79s/it][A
 62%|██████▎   | 5/8 [1:29:40<55:55, 1118.38s/it] [A
 75%|███████▌  | 6/8 [1:46:55<36:20, 1090.16s/it][A
 88%|████████▊ | 7/8 [2:06:06<18:30, 1110.07s/it][A
100%|██████████| 8/8 [2:25:50<00:00, 1133.47s/it][A100%|██████████| 8/8 [2:25:50<00:00, 1093.78s/it]
 36%|███▌      | 5/14 [13:39:04<31:28:37, 12590.86s/it]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [27:55<3:15:29, 1675.68s/it][A
 25%|██▌       | 2/8 [59:06<2:59:03, 1790.56s/it][A
 38%|███▊      | 3/8 [1:38:10<2:50:15, 2043.12s/it][A
 50%|█████     | 4/8 [2:03:48<2:02:54, 1843.67s/it][A 50%|█████     | 4/8 [2:07:44<2:07:44, 1916.10s/it]
 36%|███▌      | 5/14 [15:46:48<28:24:15, 11361.75s/it]
Traceback (most recent call last):
  File "test_el_stability.py", line 761, in <module>
    weights, feat_pos = get_linda_features(instance, cls, scaler, dataset_ref, 1, feat_list, 1)
  File "test_el_stability.py", line 257, in get_linda_features
    [bn, inference, infoBN] = generate_BN_explanations(instance, label_lst, feat_list, "Result", 
  File "/mnt/home/n9455647/full_stability/learning.py", line 692, in generate_BN_explanations
    bn, infoBN, essencGraph = learnBN( path_to_permutations )
  File "/mnt/home/n9455647/full_stability/learning.py", line 542, in learnBN
    infoBN = gnb.getInformation( bn )  
  File "/home/n9455647/.conda/envs/conda_test/lib/python3.8/site-packages/pyAgrum/lib/notebook.py", line 698, in getInformation
    return _reprInformation(bn, evs, size, cmap, asString=True)
  File "/home/n9455647/.conda/envs/conda_test/lib/python3.8/site-packages/pyAgrum/lib/notebook.py", line 640, in _reprInformation
    ie.makeInference()
  File "/home/n9455647/.conda/envs/conda_test/lib/python3.8/site-packages/pyAgrum/pyAgrum.py", line 13476, in makeInference
    return _pyAgrum.LazyPropagation_makeInference(self)
Exception: std::bad_alloc
